src/Microsoft.Management.Deployment/Helpers.cpp

```diff
@@ -143,4 +145,26 @@ namespace winrt::Microsoft::Management::Deployment::implementation
 
         return callerName;
     }
+
+    bool ShouldDelayBackgroundUpdateByCaller()

```

Comment:
I really meant with a name that was like `IsBackgroundProcessForPolicy` or something.